src/components/totals/GoalProgress/index.js

```diff
@@ -74,13 +78,25 @@ module.exports = React.createClass({
 
   renderText: function() {
     var props = this.props;
+    var message;
+
+    if(!props.goal && props.fallbackToFundsRaised) {
```

Comment:
Can this block be extracted into separate function to simplify the renderText function:

```
  getMessage: function() {
    if(!props.goal && props.fallbackToFundsRaised) {
      return this.tm('no_goal_text', {
        total: this.formatCurrency(props.total)
      });
    } else {
      ...
    }
  },
```